app/controllers/carts_controller.rb

```diff
@@ -8,14 +18,20 @@ def checkout_cart
     end
   end
 
-  def create
-    update_cart
-    flash[:notice] = 'Added to Cart'
-    redirect_to items_path
+  def delete_item
+    if params[:cart].nil?
+      delete_all_items_from_cart
+    else
+      delete_specific_item_from_cart
+      flash[:notice] = 'Item removed from cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.